lib/Backend/Commands.hs

```diff
@@ -324,6 +327,14 @@ copyCmd (CopyOptions oldPath newPath force) = do
           <&> (\(CopyOperation old new) -> (old ^. E.path, new ^. E.path) )
           & throw . CPREntryAlreadyExists
 
+    checkForEntriesInPath :: Path -> Sem r ()
+    checkForEntriesInPath path =
+      when (not (null (path ^. unPath))) do
+        let pathInits = Path.EntryPath . NE.fromList <$> (tail . inits) (path ^. unPath)
```

Comment:
This check is not enough.

Here's a counterexample:

```
$ cabal run exe:coffer -- create test/dir/entry1                                        
$ cabal run exe:coffer -- create test/dir/entry2
$ cabal run exe:coffer -- create test2/dir

$ cabal run exe:coffer -- view /

/
  test/
    dir/
      entry1 - [2022-02-24 11:48:52]
      entry2 - [2022-02-24 11:48:56]
  test2/
    dir - [2022-02-24 11:49:07]
```

```
$ cabal run exe:coffer -- copy test test2
[SUCCESS] Copied '/test/dir/entry1' to '/test2/dir/entry1'.
[SUCCESS] Copied '/test/dir/entry2' to '/test2/dir/entry2'.

$ cabal run exe:coffer -- view /
/
  test/
    dir/
      entry1 - [2022-02-24 11:48:52]
      entry2 - [2022-02-24 11:48:56]
  test2/
    dir - [2022-02-24 11:49:07]
    dir/
      entry1 - [2022-02-24 11:49:20]
      entry2 - [2022-02-24 11:49:20]
```

Now `/test2/dir` points to both a directory and an entry.

Reply:
Done